Berksfile

```diff
@@ -1,6 +1,11 @@
-cookbook "mysql", git: "https://github.com/opscode-cookbooks/mysql", tag: "v3.0.12"
-cookbook "yum", "< 3.0.0"
+source 'https://supermarket.chef.io'
 
-site :opscode
+cookbook 'database'
+# cookbook 'percona', '>= 0.15.0'
+cookbook 'percona', github: 'phlipper/chef-percona'
+cookbook 'firewall', git: 'git@github.com:osuosl-cookbooks/firewall'
+cookbook 'nagios', git: 'git@github.com:osuosl-cookbooks/nagios'
+cookbook 'osl-munin', git: 'git@github.com:osuosl-cookbooks/osl-munin'
+cookbook 'sysctl'
```

Comment:
So we're not including cookbooks in the Berksfile that are referenced in metadata anymore? I know that it will find it on the supermarket, but I thought we had been adding them anyways? If that's changed, then it'd be good to standardize.